I'm echoing to stdout
I'm echoing to stderr
My JobID is 55816610
I have 4 CPUs on node r108u25n01
Using backend: pytorch
cuda not available
[I] Loading dataset ZINC...
train, test, val sizes : 10000 1000 1000
[I] Finished loading.
[I] Data load time: 5.0861s
Training Graphs:  10000
Validation Graphs:  1000
Test Graphs:  1000
  0%|          | 0/1000 [00:00<?, ?it/s]Epoch 0:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch 0:   0%|          | 0/1000 [01:05<?, ?it/s, lr=0.001, test_MAE=1.3, time=65.8, train_MAE=1.36, train_loss=1.36, val_MAE=1.23, val_loss=1.23]Epoch 0:   0%|          | 1/1000 [01:05<18:15:38, 65.80s/it, lr=0.001, test_MAE=1.3, time=65.8, train_MAE=1.36, train_loss=1.36, val_MAE=1.23, val_loss=1.23]Epoch 1:   0%|          | 1/1000 [01:05<18:15:38, 65.80s/it, lr=0.001, test_MAE=1.3, time=65.8, train_MAE=1.36, train_loss=1.36, val_MAE=1.23, val_loss=1.23]Epoch 1:   0%|          | 1/1000 [02:04<18:15:38, 65.80s/it, lr=0.001, test_MAE=1.18, time=58.7, train_MAE=1.19, train_loss=1.19, val_MAE=1.12, val_loss=1.12]Epoch 1:   0%|          | 2/1000 [02:04<17:39:22, 63.69s/it, lr=0.001, test_MAE=1.18, time=58.7, train_MAE=1.19, train_loss=1.19, val_MAE=1.12, val_loss=1.12]Epoch 2:   0%|          | 2/1000 [02:04<17:39:22, 63.69s/it, lr=0.001, test_MAE=1.18, time=58.7, train_MAE=1.19, train_loss=1.19, val_MAE=1.12, val_loss=1.12]slurmstepd: error: *** JOB 55816610 ON r108u25n01 CANCELLED AT 2025-07-15T10:23:43 ***
